### PR TITLE
Fix contents of blobs when binary contents are specified for r11s driver

### LINKED ISSUE #7089: R11s and tinylicious 400 error while creating container.
## Describe the bug
When uploading a summary while creating a container by attach which contains blob contents of UintArray type, we get a error from server. Server is unable to process the blob contents. 

### Logs
ypeError [ERR_INVALID_ARG_TYPE]: The first argument must be of type string or an instance of Buffer, ArrayBuffer, or Array or an Array-like Object. Received an instance of Object

alfred_1       |     at Function.from (buffer.js:330:9)

alfred_1       |     at Object.Uint8ArrayToString (/usr/src/server/node_modules/@fluidframework/common-utils/dist/bufferNode.js:23:23)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryBlob (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:62:47)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryTreeObject (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:41:29)

alfred_1       |     at /usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:26:43

alfred_1       |     at Array.map (<anonymous>)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryTreeCore (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:24:73)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryTreeObject (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:50:29)

alfred_1       |     at /usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:26:43

alfred_1       |     at Array.map (<anonymous>)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryTreeCore (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:24:73)

alfred_1       |     at SummaryTreeUploadManager.writeSummaryTree (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/node_modules/@fluidframework/server-services-client/dist/summaryTreeUploadManager.js:21:21)

alfred_1       |     at runMicrotasks (<anonymous>)

alfred_1       |     at processTicksAndRejections (internal/process/task_queues.js:97:5)

alfred_1       |     at async DocumentStorage.createDocument (/usr/src/server/packages/routerlicious/node_modules/@fluidframework/server-routerlicious-base/node_modules/@fluidframework/server-services/node_modules/@fluidframework/server-services-shared/dist/storage.js:53:24)

### PR DESCRIPTION
**Pull request by jatgarg**
Fixes: https://github.com/microsoft/FluidFramework/issues/7089
Blob contents needs to be converted to string when Unint8Array is supplied to create new summary.

### CODE REVIEW

**Review by vladsud — APPROVED** [review]

File: packages/drivers/routerlicious-driver/src/createNewUtils.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Uint8ArrayToString } from "@fluidframework/common-utils";
+import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
+
+export function convertSummaryToCreateNewSummary(summary: ISummaryTree): ISummaryTree {
```
[the remark this answers is not in the document]
Comment (vladsud) [reply packages/drivers/routerlicious-driver/src/createNewUtils.ts:9]: I also would put some comment describing what and why this function does.

[the remark this answers is not in the document]
Comment (tanviraumi) [reply packages/drivers/routerlicious-driver/src/createNewUtils.ts:9]: Ditto on the comment part. Quick question: does ODSP support binary blob already?

**Review by tanviraumi — APPROVED** [review]

**Comment by jatgarg** [comment]
@tanviraumi No, there also we convert the blob contents into strings.

**Comment by tanviraumi** [comment]
> @tanviraumi No, there also we convert the blob contents into strings.

Then why is it driver specific and not something which is shared? Does r11s driver has specific edge cases?

**Comment by jatgarg** [comment]
No actually they take "IOdspSummaryTree" as create new summary while the r11s takes ISummaryTree.

File: packages/drivers/routerlicious-driver/src/createNewUtils.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jatgarg) [reply packages/drivers/routerlicious-driver/src/createNewUtils.ts:14]: Added comment. For creating a separate interface, we can follow up after this as we may want to see if that could be something we use in runtime too and then all the drivers and not just in r11s driver.

**Pull request merged**